implementations/micrometer-registry-stackdriver/src/main/java/io/micrometer/stackdriver/StackdriverMeterRegistry.java

```diff
@@ -87,6 +88,8 @@ public StackdriverMeterRegistry(StackdriverConfig config, Clock clock) {
     private StackdriverMeterRegistry(StackdriverConfig config, Clock clock, ThreadFactory threadFactory,
                                      Callable<MetricServiceSettings> metricServiceSettings) {
         super(config, clock);
+        requireNonNull(config.projectId());
```

Comment:
We should probably use the `MissingRequiredConfigurationException` class we have like at least `WavefrontMeterRegistry` does for #417.

Reply:
@shakuzen Thanks for the feedback! I created https://github.com/micrometer-metrics/micrometer/pull/1277 to resolve it against the 1.0.x branch. I'll push another commit on this PR for the 1.1.x branch.

Reply:
@shakuzen I pushed e59137b.